recipes/liburcu/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.12" %}
+
+package:
+  name: liburcu
+  version: {{ version }}
+
+source:
+  url: https://lttng.org/files/urcu/userspace-rcu-latest-{{ version }}.tar.bz2
+  sha256: bbfaead0345642b97e0de90f889dfbab4b2643a6a5e5c6bb59cd0d26fc0bcd0e
+
+build:
+  number: 0
+  skip: true  # [osx]
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX $PKG_NAME  
```

Comment:
Conda inspect is deprecated. No need to run it here b/c that is run by default now in all builds.